packages/docs/src/styles/base/_typography.scss

```diff
@@ -1,8 +1,23 @@
 blockquote {
-  background: $color-primary-alt-lightest;
-  border-left: 2px solid $color-primary-alt-light;
+  background-color: $color-gray-lightest;
+  border-left: 4px solid $color-gray-light;
+  font-family: $font-sans;
+  font-style: italic;
   margin-left: 0;
   padding: $spacer-2;
+  padding-left: $spacer-6;
+  position: relative;
+
+
+  &::before {
```

Comment:
This is a bit tricky because screen readers will read out the quotation mark and it will be an additional element to move through before reaching the quote. Normally something like this could be put in a `span` with aria hidden, but since this is just css im not sure if its possible to hide it. 
@davidakennedy 
<img width="500" alt="Screen Shot 2019-10-24 at 11 09 58 AM" src="https://user-images.githubusercontent.com/5424713/67504573-401cf680-f64f-11e9-99de-7980ae17f4b2.png">

Reply:
@line47 @bernardwang Yeah, most modern screen reader and browser pairings will announce this. While it's not ideal, it's not horrible either.

It would be great to hide it, but the only reliable way I can think of is to put the generated content on a `<span aria-hidden="true">` like that. But that can be a pain if you're just writing markdown files.

I think it's fine to leave it as is, but we should be mindful of how much we use CSS generated content in the design system and on docs so we don't overdo it.